src/Tribe/Aggregator/Record/Abstract.php

```diff
@@ -139,7 +139,14 @@ public function setup_meta( $meta ) {
 	 */
 	public function update_meta( $key, $value ) {
 		$this->meta[ $key ] = $value;
-		return update_post_meta( $this->post->ID, self::$meta_key_prefix . $key, $value );
+
+		$field = self::$meta_key_prefix . $key;
+
+		if ( null === $value ) {
```

Comment:
We only want to delete the meta if it is `null`. We'll not delete if it is a blank string or `0`.